src/ac-power/ac-power.c

```diff
@@ -57,9 +59,8 @@ static int parse_argv(int argc, char *argv[]) {
                 }
 
         if (optind < argc)
-                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
-                                       "%s takes no arguments.",
-                                       program_invocation_short_name);
+                return log_error_errno(
+                        SYNTHETIC_ERRNO(EINVAL), "%s takes no arguments.", program_invocation_short_name);
```

Comment:
No.

Reply:
may be changeable in .clang-format file, will investigate...